lib/activerecord_slotted_counters/has_slotted_counter.rb

```diff
@@ -182,9 +182,11 @@ def prepare_slotted_counters_params(ids, counters)
     end
 
     def increment!(attribute, by = 1, touch: nil)
-      super unless self.class.registered_slotted_counter? attribute
-
-      self.class.update_counters(id, attribute => by, :touch => touch)
+      if self.class.registered_slotted_counter? attribute
+        self.class.update_counters(id, attribute => by, :touch => touch)
+      else
+        super
+      end
```

Comment:
Purely stylistic (and optional) suggestion, but IMO guard clauses can be more readable and diff can be shorter if we just add `return` before call to `super` under condition:

```suggestion
      return super unless self.class.registered_slotted_counter? attribute

      self.class.update_counters(id, attribute => by, :touch => touch)
```